python_modules/dagster/dagster/_core/storage/event_log/base.py

```diff
@@ -124,6 +114,16 @@ def last_materialization_storage_id(self) -> Optional[int]:
         return self.last_materialization_record.storage_id
 
 
+class AssetPartitionRecord(NamedTuple):
```

Comment:
I think `AssetCheckSummaryRecord` reflects our latest preferred naming convention, which would suggest naming this `AssetPartitionSummaryRecord`.

Reply:
Yeah... I can't keep track of what the latest consensus was on the semantics of `Record`.

The most we got into it was in the discussion on this PR:  https://github.com/dagster-io/dagster/pull/21470